doc/user/data/mz_now/mz_now_combination.yml

```diff
@@ -0,0 +1,39 @@
+columns:
```

Comment:
This is used in the single-sourced details about mz_now clauses; for example:

https://preview.materialize.com/materialize/33041/transform-data/idiomatic-materialize-sql/mz_now/#disjunctions-or